Apps/Contoso.Android.Puppet/ModulePages/AppCenterFragment.cs

```diff
@@ -114,7 +114,7 @@ private void LogWriteLevelClicked(object sender, EventArgs e)
 
         private void UserIdTextKeyPressedHandler(object sender, View.KeyEventArgs e)
         {
-            if (e.Event.Action == KeyEventActions.Down && e.KeyCode == Keycode.Enter)
+            if ((e.Event.Action == KeyEventActions.Down && e.KeyCode == Keycode.Enter) || (e.Event.Action == KeyEventActions.Up))
```

Comment:
The first condition became redundant now

Reply:
Yeah, You're right, Thank you!